### PR TITLE
Fix notification wording

### PR DESCRIPTION
**Pull request by Flash1232**

### CODE REVIEW

File: app/src/main/java/org/grapheneos/apps/client/service/SeamlessUpdaterJob.kt [hunk 1]
```diff
@@ -74,13 +74,13 @@ class SeamlessUpdaterJob : JobService() {
 
                 if (requireConfirmation.isNotBlank() && result.requireConfirmation.isNotEmpty()) {
                     if (content.isNotBlank()) content += ", "
-                    content += "$requireConfirmation update update available."
+                    content += "$requireConfirmation: update available!"
```
Comment (thestinger) [inline app/src/main/java/org/grapheneos/apps/client/service/SeamlessUpdaterJob.kt:77]: I don't think we should have any of these exclamation marks in any of the notifications. I know there are some messages that have that but it should be removed.

Comment (Flash1232) [reply app/src/main/java/org/grapheneos/apps/client/service/SeamlessUpdaterJob.kt:77]: I'll remove some exclamation marks alongside then!

**Comment by thestinger** [comment]
This conflicts with the current code in the main branch so you'll need to resolve that.

**Comment by thestinger** [comment]
It needs to be rebased on top of the latest upstream code. It still conflicts.

**Comment by Flash1232** [comment]
Hmm I'm sorry that you'll always get notified while I try to figure it out. Can't make it a "draft" ..

**Comment by thestinger** [comment]
It can be made into a draft after creating it. I did that for you but you could have. It's shown as a link on the side in the web UI.

**Comment by Flash1232** [comment]
I totally expected it to be somewhere else. Thanks for helping me out on that :)

**Pull request merged**